scripts/JSRootCore.js

```diff
@@ -1949,15 +1949,15 @@
             return this._func(x, y);
          }
          m.GetParName = function(n) {
-            if (this.fParams && this.fParams.fParNames) return this.fParams.fParNames[n];
-            if (this.fFormula && this.fFormula.fParams) return this.fFormula.fParams[n].first;
+	    if (this.fFormula && this.fFormula.fParams) return this.fFormula.fParams[n].first;
             if (this.fNames && this.fNames[n]) return this.fNames[n];
-            return "p"+n;
+            if (this.fParams.fParNames[n]!==undefined) return this.fParams.fParNames[n];
```

Comment:
You should check that this.fParams exists - see my fixes to your previous PR.
``` 
 if (this.fParams && this.fParams.fParNames) return this.fParams.fParNames[n];
```
Please fix it before I can merge